class_person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_accessor :age, :name, :parent_permission
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # Private method of_age? that returns true if @age is greater or equal to 18 and false otherwise.
+
+  private
+
+  def of_age
+    age >= 18
+  end
+
+  # Public method can_use_services? that returns true if person is of age or if they have permission from parents.
+  def can_use_services
+    of_age? || @parent_permission
+  end
```

Comment:
The "canuse_services" method is a public one. the keyword "private" relates to the "of_age?" method.

Reply:
- [ ] 👍 @diarisdiakite, Hi! The `can_use_services` method is currently marked as private, which means it can't be accessed from outside the class. To make it accessible externally, please consider changing its access modifier to public. This way, other parts of your code can utilize this method as intended. Keep up the great work! 🚀

Check the example bellow:
```
person = Person.new(10, 22, name='maximilianus')
person.can_use_services

```

![image](https://github.com/diarisdiakite/oop_school-library/assets/55840999/9749c3fe-32af-4057-b557-268c360946d5)